app/.helper/package

```diff
@@ -78,6 +78,11 @@ get_package_platform() {
         sed_opts="-nE"
     fi
 
+    # macOS with silicon chips need to download the darwin-x86-64 variant
+    if [[ $(uname -m) == 'arm64' ]]; then
```

Comment:
Is there any other identifier that we can use to specifically target the macOS arm64?
If we only check by machine hardware type, if in the future we're using Linux arm64, this will break the package installation for Linux.

Reply:
I think you could use `uname -o` == 'Darwin' for mac